db/seeds/data/locations.json

```diff
@@ -0,0 +1,7 @@
+[{
+  "name": "Toronto"
+},
+{
+  "name": "New York"
+}
+]
```

Comment:
Nit: can you format it as seen [here](https://tools.ietf.org/html/rfc4627#page-8). I know the movies seed data isn't like this, but it just makes it easier to read when it's formatted correctly.

```
[
  {
    "name": "Toronto"
  },
  {
    "name": "New York"
  }
]
```